Sources/PointFree/Account/Account.swift

```diff
@@ -249,29 +264,47 @@ private let subscriptionInviteMoreRowView = View<Prelude.Unit> { _ in
     ])
 }
 
-private let subscriptionPaymentInfoView = View<Prelude.Unit> { _ in
-  gridRow([`class`([subscriptionInfoRowClass])], [
-    gridColumn(sizes: [.mobile: 3, .desktop: 2], [
-      div([
-        p(["Payment"])
-        ])
-      ]),
-    gridColumn(sizes: [.mobile: 9, .desktop: 5], [
-      div([`class`([Class.padding([.mobile: [.leftRight: 1]])])], [
-        p(["American Express 3*** ****** *2002"]),
-        p(["Expiration: 1/2018"]),
-        p(["Next payment due: 2017-12-24"]),
-        p(["Total Amount: $18.00"])
-        ])
-      ]),
-    gridColumn(sizes: [.mobile: 12, .desktop: 5], [
-      div([`class`([Class.grid.end(.mobile)])], [
-        p([
-          a([href("#"), `class`([Class.pf.components.button(color: .purple, size: .small)])], ["Update payment method"])
+private let subscriptionPaymentInfoView = View<Stripe.Subscription> { subscription -> [Node] in
+  guard let card = subscription.customer.sources.data.first else { return [] }
+
+  return [
+    gridRow([`class`([subscriptionInfoRowClass])], [
+      gridColumn(sizes: [.mobile: 3, .desktop: 2], [
+        div([
+          p(["Payment"])
+          ])
+        ]),
+      gridColumn(sizes: [.mobile: 9, .desktop: 5], [
+        div([`class`([Class.padding([.mobile: [.leftRight: 1]])])], [
+          p([text(card.brand.rawValue + " ending in " + String(card.last4))]),
+          p([text("Expires: " + String(card.expMonth) + "/" + String(card.expYear))]),
+          p([text("Next payment due: " + (subscription.currentPeriodEnd.map(dueDateFormatter.string) ?? "Never"))]),
+          p([text("Total amount: " + (totalAmount(for: subscription) ?? "Nothing"))]),
+          ])
+        ]),
+      gridColumn(sizes: [.mobile: 12, .desktop: 5], [
+        div([`class`([Class.grid.end(.mobile)])], [
+          p([
+            a([href("#"), `class`([Class.pf.components.button(color: .purple, size: .small)])], ["Update payment method"])
+            ])
           ])
         ])
       ])
-    ])
+  ]
+}
+
+private let dueDateFormatter = DateFormatter()
+  |> \.dateStyle .~ .short
+  |> \.timeStyle .~ .none
+  |> \.timeZone .~ TimeZone(secondsFromGMT: 0)
```

Comment:
I guess Linux runs in a diff time zone! We should prob figure out what to do for this but it can wait.